### PR TITLE
Enable winbind

### PR DESCRIPTION
**Pull request by gsanchietti**
Winbind is necessary since Samba 4.8.3

These changes fix the problem on a dev machine but on a production machine I still have this error:
```
session setup failed: NT_STATUS_NO_MEMORY
```
Further tests are needed.

NethServer/dev#5647

### CODE REVIEW

File: nethserver-samba.spec [hunk 1]
```diff
@@ -12,7 +12,8 @@ Obsoletes: nethserver-ibays
 Requires: samba
 Requires: tdb-tools
 Requires: nethserver-ibays
-Requires: sssd-libwbclient
+Obsoletes: sssd-libwbclient
```
Comment (DavidePrincipi) [inline nethserver-samba.spec]: I'd remove the package in an action to avoid YUM issues: the nethserver-samba RPM actually doesn't _obsolete_ anything.

Comment (gsanchietti) [reply nethserver-samba.spec]: You can't access the RPM database inside the update event.
Already tried, already failed.

Comment (DavidePrincipi) [reply nethserver-samba.spec]: Can we fix it with the "alternative" command? IIRC it is a just a symlink to a shared library

**Comment by DavidePrincipi** [comment]
Please preserve the commit history: do  **NOT SQUASH & MERGE**

**Pull request merged**